eo-runtime/src/main/rust/eo_env/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "eo_env"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+jni = "0.21.1"
+
+[lib]
+path = "src/eo_env.rs"
```

Comment:
@levBagryansky Can you help me to understand why do this `Cargo.toml` is duplicate here and in `eo-maven-plugin/src/test/resources/org/eolang/maven/binarize/eo_env/Cargo.toml`?

Reply:
@Graur `eo-runtime` and `eo-maven-plugin` formally different projects. I decided its better to specify other rust crate for `eo-maven-plugin` which will be much simplier since we check only compilability here.

Reply:
@Graur does not make the `eo-maven-plugin` dependent on the `eo-runtime`

Reply:
@levBagryansky I see, thanks